cmd/utils/flags.go

```diff
@@ -1509,7 +1509,7 @@ func setMiner(ctx *cli.Context, cfg *miner.Config) {
 		cfg.Recommit = ctx.Duration(MinerRecommitIntervalFlag.Name)
 	}
 	if ctx.IsSet(MinerNewPayloadTimeout.Name) {
-		cfg.NewPayloadTimeout = ctx.Duration(MinerNewPayloadTimeout.Name)
```

Comment:
Should we deprecate `NewPayloadTimeout` flag?

Originally 
- `Recommit` is used to specify the interval for re-creating pending block
- `NewPayloadTimeout` is used to specify the maximum time allowance for creating a single pending block

I agree that these two flags are kind of overlapped, especially after getting rid of the PUSH mode in miner.